logstash-core-event-java/src/main/java/com/logstash/ext/JrubyEventExtLibrary.java

```diff
@@ -233,11 +215,7 @@ public IRubyObject ruby_sprintf(ThreadContext context, IRubyObject format) throw
             try {
                 return RubyString.newString(context.runtime, event.sprintf(format.toString()));
             } catch (IOException e) {
-                throw new RaiseException(
-                    getRuntime(),
-                    (RubyClass) getRuntime().getModule("LogStash").getClass("Error"),
-                    "timestamp field is missing", true
-                );
+                throw new RaiseException(getRuntime(), (RubyClass)getRuntime().getModule("LogStash").getClass("Error"), "timestamp field is missing", true);
```

Comment:
I will actually create a followup issue to review all exception handling in the Java code